client/src/components/formHelpers/FormValidators.js

```diff
@@ -0,0 +1,29 @@
+// Matches editor links for: Repl.it, Glitch, CodeSandbox
+const editorRegex = /repl\.it\/@|edit\/#!|codesandbox\.io\/s\//;
```

Comment:
We could add the domain for the glitch check just to avoid confusion. ( completely your call)

Reply:
The `edit/#!` is Glitch